src/App.jsx

```diff
@@ -14,57 +15,67 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectedGood, setSelectedGood] = useState('Jam');
 
-    <h1 className="title is-flex is-align-items-center">
-      Jam is selected
-      <button data-cy="ClearButton" type="button" className="delete ml-3" />
-    </h1>
+  const handleSelectGood = good => {
```

Comment:
The `handleSelectGood` function should be split into two separate functions for adding and removing selections for better readability.

Reply:
Example:
```
  const selectGood = (good) => {
    setSelectedGood(good);
  };

  const clearSelection = () => {
    setSelectedGood('');
  };
```